include/dsn/dist/replication/replication_app_base.h

```diff
@@ -268,9 +273,10 @@ class replication_app_base : public replica_base
     ::dsn::error_code update_init_info_ballot_and_decree(replica *r);
 
 protected:
-    std::string _dir_data;   // ${replica_dir}/data
-    std::string _dir_learn;  // ${replica_dir}/learn
-    std::string _dir_backup; // ${replica_dir}/backup
+    std::string _dir_data;      // ${replica_dir}/data
+    std::string _dir_learn;     // ${replica_dir}/learn
+    std::string _dir_backup;    // ${replica_dir}/backup
+    std::string _dir_bulk_load; // ${replica_dir}/.bulk_load
```

Comment:
It seems we missed this naming problem in our previous review. I recommend not to prefix a dot before the directory. Better to keep the directories naming aligned.